test/integration/typeform-translate.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { channelsPlugin } from '@balena/jellyfish-plugin-channels';
+import { defaultPlugin } from '@balena/jellyfish-plugin-default';
+import { productOsPlugin } from '@balena/jellyfish-plugin-product-os';
+import { testUtils } from '@balena/jellyfish-worker';
+import _ from 'lodash';
+import path from 'path';
+import { typeformPlugin } from '../../lib';
+import webhooks from './webhooks';
+
+let ctx: testUtils.TestContext;
+
+beforeAll(async () => {
+	ctx = await testUtils.newContext({
+		plugins: [
+			defaultPlugin(),
+			productOsPlugin(),
+			channelsPlugin(),
+			typeformPlugin(),
+		],
+	});
+
+	await testUtils.translateBeforeAll(ctx);
+}, 10000);
```

Comment:
This timeout increase seems to only be necessary for balenaCI. Tests ran locally or in local compose pass everytime with no issues.